modules/govuk/files/etc/govuk/import_mapit_data.sh

```diff
@@ -17,11 +17,24 @@ function restart_mapit_services () {
   sudo service nginx start
 }
 
-stop_mapit_services
-# This script needs to run as a user that can sudo to become the postgres
-# user.  It's easiest if this is passwordless, so we can't use the deploy user
-# who can't sudo without a password
-# Run 'govuk_setenv mapit ./import-db-from-s3.sh' from the '/var/apps/mapit' as
 
-cd /var/apps/mapit && govuk_setenv mapit ./import-db-from-s3.sh
+function import_db_from_s3 () {
+  # get the data from s3
+  curl 'https://s3-eu-west-1.amazonaws.com/govuk-custom-formats-mapit-storage-production/source-data/2020-11/mapit-november-2020-update.sql.gz' -o /tmp/mapit.sql.gz
+  if ! echo "e6d7d0049348744cf4c8c2f3f921adfcc0a3c76d /tmp/mapit.sql.gz" | sha1sum -c -; then
```

Comment:
this sha will be a pain to update when the data changes. It'd be great if we could figure out a way for this not to be reliant on a puppet deployment. (Appreciate that you're fixing a thing right now :-) )

Reply:
Probably just remove the checksum then? Not sure it's really adding any value.

Reply:
I guess there are 3 options for the checksum:
1. when someone wants to update the URL of the S3 object database, they update the checksum too or
2. we remove the checksum like Chris said
3. we retrieve the checksum from s3 and compare it with the locally computed one if we are afraid the s3 object is corrupted on download?

On the second issue of not being reliant on Puppet, this is the easiest method, we could transfer everything in the capistrano deployment but the `deploy` user there is limited.

Reply:
The download is already over HTTPS from a bucket which only we can write to, and the chance of corruption is negligible because this will run maybe a few times a month at most. If you're worried about removing the checksum, don't be :)